test/internal/kubernetes/utils_create.go

```diff
@@ -30,21 +47,18 @@ type cnsScenario struct {
 }
 
 const (
-	envTestDropgz              = "TEST_DROPGZ"
-	envCNIDropgzVersion        = "CNI_DROPGZ_VERSION"
-	envCNSVersion              = "CNS_VERSION"
-	EnvInstallCNS              = "INSTALL_CNS"
-	envInstallAzilium          = "INSTALL_AZILIUM"
-	envInstallAzureVnet        = "INSTALL_AZURE_VNET"
-	envInstallOverlay          = "INSTALL_OVERLAY"
-	envInstallAzureCNIOverlay  = "INSTALL_AZURE_CNI_OVERLAY"
-	envInstallDualStackOverlay = "INSTALL_DUALSTACK_OVERLAY"
-	cnsLabelSelector           = "k8s-app=azure-cns"
+	envTestDropgz           = "TEST_DROPGZ"
+	envCNIDropgzVersion     = "CNI_DROPGZ_VERSION"
+	envCNSVersion           = "CNS_VERSION"
+	EnvInstallCNS           = "INSTALL_CNS"
+	cnsLinuxLabelSelector   = "k8s-app=azure-cns"
+	cnsWindowsLabelSelector = "k8s-app=azure-cns-win"
 )
 
 var (
 	ErrUnsupportedCNSScenario = errors.New("unsupported CNS scenario")
 	ErrPathNotFound           = errors.New("failed to get the absolute path to directory")
+	ErrNoCNSScenarioDefined   = errors.New("no CNSScenario set to true as env var")
```

Comment:
Should this be changed to `CNSScenario is not defined`? Would read out `no CNSSCenario env vars set to true, must explicitly set one to true: CNSScenario is not defined` instead of `no CNSSCenario env vars set to true, must explicitly set one to true: no CNSScenario set to true as env var`